tests/cpu/Baker_tests.cpp

```diff
@@ -106,63 +92,361 @@ OCIO_ADD_TEST(Baker, bake)
+    bake->setDisplayView("sRGBInactive", "Film");
+    bake->setFormat("cinespace");
+
+    os.str("");
+    OCIO_CHECK_NO_THROW(bake->bake(os));
+
+    // Shared view.
+    bake = OCIO::Baker::Create();
+    bake->setConfig(config);
+    bake->setInputSpace("Raw");
+    bake->setDisplayView("sRGB", "Raw");
+    bake->setFormat("cinespace");
+
+    os.str("");
+    OCIO_CHECK_NO_THROW(bake->bake(os));
+
+    // Inactive view.
+    bake = OCIO::Baker::Create();
+    bake->setConfig(config);
+    bake->setInputSpace("Raw");
+    bake->setDisplayView("sRGB", "FilmInactive");
+    bake->setFormat("cinespace");
+
+    os.str("");
+    OCIO_CHECK_NO_THROW(bake->bake(os));
+
+    // Inactive shared view.
+    bake = OCIO::Baker::Create();
+    bake->setConfig(config);
+    bake->setInputSpace("Raw");
+    bake->setDisplayView("sRGBInactive", "RawInactive");
+    bake->setFormat("cinespace");
+
+    os.str("");
+    OCIO_CHECK_NO_THROW(bake->bake(os));
+
+    // Baking 1D LUT with Crosstalk.
+    bake = OCIO::Baker::Create();
+    bake->setConfig(config);
+    bake->setInputSpace("Raw");
+    bake->setDisplayView("sRGB", "Film");
+    bake->setFormat("spi1d");
+
+    os.str("");
+    OCIO_CHECK_THROW_WHAT(bake->bake(os), OCIO::Exception, "The format 'spi1d' does not support transformations with channel crosstalk.");
+
+    // Cube Size < 2.
+    bake = OCIO::Baker::Create();
+    bake->setConfig(config);
+    bake->setInputSpace("Raw");
+    bake->setTargetSpace("sRGB");
+    bake->setCubeSize(1);
+    bake->setFormat("cinespace");
+
+    os.str("");
+    OCIO_CHECK_THROW_WHAT(bake->bake(os), OCIO::Exception, "Cube size must be at least 2 if set.");
+
+    // Shaper Size < 2.
+    bake = OCIO::Baker::Create();
+    bake->setConfig(config);
+    bake->setInputSpace("Raw");
+    bake->setTargetSpace("sRGB");
+    bake->setShaperSpace("Log");
+    bake->setShaperSize(1);
+    bake->setFormat("resolve_cube");
+
+    os.str("");
+    OCIO_CHECK_THROW_WHAT(bake->bake(os), OCIO::Exception, "A shaper space 'Log' has been specified, so the shaper size must be 2 or larger.");
+
+    // Using shaper with unsupported format.
+    bake = OCIO::Baker::Create();
+    bake->setConfig(config);
+    bake->setInputSpace("Raw");
+    bake->setTargetSpace("sRGB");
+    bake->setShaperSpace("Log");
+    bake->setFormat("iridas_itx");
+
+    os.str("");
+    OCIO_CHECK_THROW_WHAT(bake->bake(os), OCIO::Exception, "The format 'iridas_itx' does not support shaper space.");
+
+    // Using shaper space with Crosstalk.
+    bake = OCIO::Baker::Create();
+    bake->setConfig(config);
+    bake->setInputSpace("Raw");
+    bake->setTargetSpace("sRGB");
+    bake->setShaperSpace("Crosstalk");
+    bake->setFormat("cinespace");
+
+    os.str("");
+    OCIO_CHECK_THROW_WHAT(bake->bake(os), OCIO::Exception, "The specified shaper space, 'Crosstalk' has channel crosstalk, which is not appropriate for shapers. Please select an alternate shaper space or omit this option.");
```

Comment:
There is a limit of 100 characters per line.

Reply:
Done.